### ISSUE TITLE
Bug: "Could not connect: Connection timed out" error...

### ISSUE THREAD

**Issue by tuqueque** [mattermost/focalboard#3826]
Hello, I installed Focalboard from the AUR (https://aur.archlinux.org/packages/focalboard-bin ) repository (Manjaro Linux), but when I open the app, the window is just blank for about 1 minute and after that, it turns white with a prompt that says:
`Could not connect: Connection timed out`
![2022-09-11_00-56](https://user-images.githubusercontent.com/13471025/189513977-28621cf0-55e3-4d52-b4fb-210d4a3e7dd8.png)

I tried downloading the app from here, ran it through the Terminal, same result... here's what I got in the Terminal:
```
./focalboard-app
Overriding existing handler for signal 10. Set JSC_SIGNAL_FOR_GC if you want WebKit to use a different signal
2022/09/11 01:10:04 ==  init: migrating  ===============================================================================
2022/09/11 01:10:04 ==  init: migrated (0.0185s)  ======================================================================
2022/09/11 01:10:04 ==  system_settings_table: migrating  ==============================================================
2022/09/11 01:10:04 ==  system_settings_table: migrated (0.0235s)  =====================================================
2022/09/11 01:10:04 ==  blocks_rootid: migrating  ======================================================================
2022/09/11 01:10:04 ==  blocks_rootid: migrated (0.0264s)  =============================================================
2022/09/11 01:10:04 ==  auth_table: migrating  =========================================================================
2022/09/11 01:10:04 ==  auth_table: migrated (0.0235s)  ================================================================
2022/09/11 01:10:04 ==  blocks_modifiedby: migrating  ==================================================================
2022/09/11 01:10:04 ==  blocks_modifiedby: migrated (0.0348s)  =========================================================
2022/09/11 01:10:04 ==  sharing_table: migrating  ======================================================================
2022/09/11 01:10:04 ==  sharing_table: migrated (0.0333s)  =============================================================
2022/09/11 01:10:04 ==  workspaces_table: migrating  ===================================================================
2022/09/11 01:10:04 ==  workspaces_table: migrated (0.0333s)  ==========================================================
2022/09/11 01:10:04 ==  teams: migrating  ==============================================================================
2022/09/11 01:10:04 ==  teams: migrated (0.0404s)  =====================================================================
2022/09/11 01:10:04 ==  blocks_history: migrating  =====================================================================
2022/09/11 01:10:04 ==  blocks_history: migrated (0.0445s)  ============================================================
2022/09/11 01:10:04 ==  blocks_created_by: migrating  ==================================================================
2022/09/11 01:10:04 ==  blocks_created_by: migrated (0.0442s)  =========================================================
2022/09/11 01:10:04 ==  match_collation: migrating  ====================================================================
2022/09/11 01:10:04 ==  match_collation: migrated (0.0458s)  ===========================================================
2022/09/11 01:10:04 ==  match_column_collation: migrating  =============================================================
2022/09/11 01:10:04 ==  match_column_collation: migrated (0.1045s)  ====================================================
2022/09/11 01:10:04 ==  millisecond_timestamps: migrating  =============================================================
2022/09/11 01:10:04 ==  millisecond_timestamps: migrated (0.0192s)  ====================================================
2022/09/11 01:10:04 ==  add_not_null_constraint: migrating  ============================================================
2022/09/11 01:10:04 ==  add_not_null_constraint: migrated (0.0195s)  ===================================================
2022/09/11 01:10:04 ==  blocks_history_no_nulls: migrating  ============================================================
2022/09/11 01:10:04 ==  blocks_history_no_nulls: migrated (0.0269s)  ===================================================
2022/09/11 01:10:04 ==  subscriptions_table: migrating  ================================================================
2022/09/11 01:10:04 ==  subscriptions_table: migrated (0.0317s)  =======================================================
2022/09/11 01:10:04 ==  add_file_info: migrating  ======================================================================
2022/09/11 01:10:04 ==  add_file_info: migrated (0.0346s)  =============================================================
2022/09/11 01:10:04 ==  add_teams_and_boards: migrating  ===============================================================
2022/09/11 01:10:04 ==  add_teams_and_boards: migrated (0.0666s)  ======================================================
2022/09/11 01:10:04 ==  populate_categories: migrating  ================================================================
2022/09/11 01:10:05 ==  populate_categories: migrated (0.0445s)  =======================================================
2022/09/11 01:10:05 ==  populate_category_blocks: migrating  ===========================================================
2022/09/11 01:10:05 ==  populate_category_blocks: migrated (0.0444s)  ==================================================
2022/09/11 01:10:05 ==  create_boards_members_history: migrating  ======================================================
2022/09/11 01:10:05 ==  create_boards_members_history: migrated (0.0438s)  =============================================
2022/09/11 01:10:05 ==  create_default_board_role: migrating  ==========================================================
2022/09/11 01:10:05 ==  create_default_board_role: migrated (0.1221s)  =================================================
2022/09/11 01:10:05 ==  persist_category_collapsed_state: migrating  ===================================================
2022/09/11 01:10:05 ==  persist_category_collapsed_state: migrated (0.0180s)  ==========================================
2022/09/11 01:10:05 ==  mark_existsing_categories_collapsed: migrating  ================================================
2022/09/11 01:10:05 ==  mark_existsing_categories_collapsed: migrated (0.0193s)  =======================================
2022/09/11 01:10:05 ==  indexes_update: migrating  =====================================================================
2022/09/11 01:10:05 ==  indexes_update: migrated (0.0307s)  ============================================================
CONSOLE JS ERROR SecurityError: The operation is insecure.
```
I would really like to test and use this application. It seems very promising!

**Comment by tuqueque** [comment]
> @tuqueque Can you try downloading and installing from our [official release page](https://github.com/mattermost/focalboard/releases) and see if you're still having issues? Can you also provide more details on your OS version and edition? Thanks! cc @sbishel

I tried both, the AUR repository and an official release from here (your Github page) and both gave me the exact same error I posted.

I use Manjaro Linux (XFCE). I keep it up to date always. Kernel 5.19...

The error I get seems to imply that is trying to establish some sort of connection. I have Internet connection, but as far as I know, this program runs locally on the computer, no Internet connection required. Is that correct?
